src/test/scala/com/campudus/tableaux/LinkTest.scala

```diff
@@ -37,6 +37,19 @@ class LinkTest extends TableauxTestBase {
   }
 
   @Test
+  def createLinkColumnWithOrdering(): Unit = okTest {
+    val postLinkColWithOrd = Json.obj("columns" -> Json.arr(Json.obj("name" -> "Test Link 1", "kind" -> "link", "fromColumn" -> 1, "toTable" -> 2, "toColumn" -> 1, "ordering" -> 5)))

```

Comment:
My mistake here. I had the impression it was the same as the ones where you change stuff. No need to edit here.